static/chats.js

```diff
@@ -52,20 +52,22 @@ Klase, kas satur visu vienas ziņas saturu, struktūru un metainformāciju
 Inicializē ar no servera atgrieztā json objekta vienu rindu
 */
 class Zinja {
-  constructor(vards, zinja) {
+  constructor(vards, zinja, laiks) {
     this.vards = vards;
     this.zinja = zinja;
+    this.laiks = laiks;
   }
 
   formateRindu() {
     this.vards = getCookie('name')
+    const laiks = this.laiks ? this.laiks : '-';
```

Comment:
šeit inicializējām mainīgo laiks, attiecīgi zemāk izmantojam vienkārši laiks. Varbūt sākumā labāk saukt citādāk nedaudz, lai rodas izpratne ātrāk.
Ar this piekļūst pie klases instances mainīgajiem, tos mēs redzam inicializētus augstāk konstruktorā.